ui/src/components/TrafficPage.tsx

```diff
@@ -232,6 +236,23 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({onTLSDetected, setAnaly
         }
     }
 
+    useEffect(() => {
```

Comment:
all of this useEffect job is to set the serviceMapStatus but it seems like you don't use this serviceMapStatus.
Please check if you need this status - it you are, use it whereever you need
if you don't need it  - please remove this useEffect & the serviceMapStatus API & serviceMapStatus state

Reply:
we don't need it anymore, we can use the window["isServiceMapEnabled"]